homeassistant/components/squeezebox/sensor.py

```diff
@@ -0,0 +1,122 @@
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:album",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:account-music",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_DURATION,
+        state_class=SensorStateClass.TOTAL,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_GENRES,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:drama-masks",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_SONGS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:file-music",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_LASTSCAN,
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_NEWPLUGINS,
+        entity_registry_visible_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_NEWVERSION,
+        entity_registry_visible_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_PLAYER_COUNT,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:folder-play",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_OTHER_PLAYER_COUNT,
+        state_class=SensorStateClass.TOTAL,
+        entity_registry_visible_default=False,
+        icon="mdi:folder-play-outline",
+    ),
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: SqueezeboxConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Platform setup using common elements."""
+
+    async_add_entities(
+        ServerStatusSensor(entry.runtime_data.coordinator, description)
+        for description in SENSORS
+    )
+
+
+class ServerStatusSensor(LMSStatusEntity, SensorEntity):
+    """LMS Status based sensor from LMS via cooridnatior."""
+
+    def __init__(
+        self,
+        coordinator: LMSStatusDataUpdateCoordinator,
+        description: SensorEntityDescription,
+    ) -> None:
+        """Initialize the sensor using description, and coordinator data."""
+        super().__init__(coordinator, description)
+        self.native_value = coordinator.data[description.key]
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self.native_value = self.coordinator.data[self.entity_description.key]
+        _LOGGER.debug(
+            "Update Sensor %s=%s", self.entity_description.key, self.native_value
+        )
+        self.async_write_ha_state()
```

Comment:
I would suggest overwriting the native_value property instead

Reply:
Oh yeah much better